examples/pxScene2d/src/rpc/rtRemoteClient.cpp

```diff
@@ -143,6 +142,7 @@ rtRemoteClient::onStreamStateChanged(std::shared_ptr<rtRemoteStream> const& /*st
 rtError
 rtRemoteClient::setStateChangedHandler(StateChangedHandler handler, void* argp)
```

Comment:
this can be called from the destructor, where we're holding mutex, so had to change the mutex type to recursive